Core/src/SplineInterpretation/SplineInterpretation.cpp

```diff
@@ -263,15 +263,15 @@ std::vector<carve::geom::vector<3>> OpenInfraPlatform::Core::SplineInterpretatio
 
 		return points;
 	}
-	catch (const std::exception& e)
+	catch (const std::exception&)
 	{
 		// in case of an error, return an empty vector
 		std::vector<carve::geom::vector<3>> pointsEmpty;
 		return pointsEmpty;
 	}

```

Comment:
It is advisable to at least log that an error occured. Use sth similiar to:

    BLUE_LOG(warning) << "error occured ... ";

Be as specific as possible, so that the future programmer knows where to look.

Reply:
Done in commit 643376a.